ethartifact/ethartifact.go

```diff
@@ -70,6 +70,54 @@ type RawArtifact struct {
 	DeployedBytecode string          `json:"deployedBytecode"`
 }
 
+type FoundryRawArtifact struct {
+	ABI      json.RawMessage `json:"abi"`
+	Bytecode struct {
+		Object string `json:"object"`
+	} `json:"bytecode"`
+	DeployedBytecode struct {
+		Object string `json:"object"`
+	} `json:"deployedBytecode"`
+	Metadata struct {
+		Settings struct {
+			CompilationTarget map[string]string `json:"compilationTarget"`
+		} `json:"settings"`
+	} `json:"metadata"`
+}
+
+func ParseFoundryArtifactFile(path string) (FoundryRawArtifact, error) {
```

Comment:
If you use this and receive ``FoundryRawArtifact`` you have no way to go to ``RawArtifact``?

I suggest you move ``convertFoundryArtifactToRawArtifact`` to ``FoundryArtifact`` and rename it to ``ToRawArtifact``.

Reply:
Good thinking! Updated